doc/man7/ssl.pod

```diff
@@ -42,9 +42,9 @@ structures:
 
 =item B<SSL_METHOD> (SSL Method)
 
-That's a dispatch structure describing the internal B<ssl> library
+That is a dispatch structure describing the internal B<ssl> library
```

Comment:
Oof, starting these item descriptions with "That is" is a very painful style ... but treat this comment as a side note for the present purpose.